test/dlc_tests/conftest.py

```diff
@@ -409,7 +410,8 @@ def is_neuron_image(fixtures):
     :param request.fixturenames: active fixtures in the request
     :return: bool
     """
-    neuron_fixtures = ["tensorflow_inference_neuron", "mxnet_inference_neuron", "pytorch_inference_neuron"]
+    neuron_fixtures = ["tensorflow_inference_neuron", "tensorflow_inference_neuronx",
```

Comment:
This conflicts with neuron_fixtures in this PR : https://github.com/aws/deep-learning-containers/pull/2776.
I am sure you are aware of it.

Reply:
Yes, I'll rebase one of the PRs once the other gets merged.